From b11f8060df869d61b8e498195857888aaa5ee51a Mon Sep 17 00:00:00 2001
From: "Kirill A. Korinsky" <kirill@korins.ky>
Date: Thu, 1 Dec 2022 21:57:34 +0100
Subject: [PATCH] Link shared metis lib against GKlib

---
 libmetis/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libmetis/CMakeLists.txt b/libmetis/CMakeLists.txt
index fc6cec6..758835f 100644
--- a/libmetis/CMakeLists.txt
+++ b/libmetis/CMakeLists.txt
@@ -7,6 +7,10 @@ file(GLOB metis_sources *.c)
 # Build libmetis.
 add_library(metis ${METIS_LIBRARY_TYPE} ${metis_sources})
 
+if(SHARED)
+  target_link_libraries(metis GKlib)
+endif()
+
 if(METIS_INSTALL)
   install(TARGETS metis
     LIBRARY DESTINATION lib
